modules/keycloak/database.tf

```diff
@@ -22,23 +22,24 @@ resource "random_string" "snapshot_prefix" {
 }
 
 resource "aws_rds_cluster" "keycloak_database" {
-  cluster_identifier_prefix = "keycloak-db-${var.environment}"
-  engine                    = "aurora"
-  engine_mode               = "serverless"
-  engine_version            = "5.6.10a"
-  availability_zones        = var.database_availability_zones
-  database_name             = "keycloak"
-  master_username           = "keycloak_admin"
-  master_password           = random_password.password.result
-  final_snapshot_identifier = "user-db-final-snapshot-${random_string.snapshot_prefix.result}-${var.environment}"
-  vpc_security_group_ids    = aws_security_group.database.*.id
-  db_subnet_group_name      = aws_db_subnet_group.user_subnet_group.name
-
+  cluster_identifier_prefix       = "keycloak-db-postgres-${var.environment}"
+  engine                          = "aurora-postgresql"
+  engine_version                  = "11.6"
+  availability_zones              = var.database_availability_zones
+  database_name                   = var.app_name
+  master_username                 = "api_admin"
```

Comment:
Should this be `keycloak_admin`?

Reply:
It should, can you tell I copied and pasted from the consignment api one :smile: